tests/ArgTest.php

```diff
@@ -1,6 +1,7 @@
 <?php
 namespace Psalm\Tests;
 
+use Person;
```

Comment:
Not sure how that will play out, this is not an existing class

Reply:
Sorry for my carelessness. Phpstorm auto import)
Fixed.